core/block_svg.js

```diff
@@ -1262,19 +1260,15 @@ Blockly.BlockSvg.prototype.updateColour = function() {
  * Enable or disable a block.
  */
 Blockly.BlockSvg.prototype.updateDisabled = function() {
-  var hasClass = Blockly.hasClass_(/** @type {!Element} */ (this.svgGroup_),
-                                   'blocklyDisabled');
```

Comment:
A separate hasClass function was inefficient since add/removeClass already do this check.